cpp-terminal/base.cpp

```diff
@@ -1,95 +1,427 @@
 #include <cpp-terminal/base.hpp>
```

Comment:
Maybe it would be easier and cleaner to have a Color header and cpp ?

Reply:
That would be possible, yeah. The original concept for the files was that "base" basically includes all function for people that want to manage the Terminal "raw" while "window" offers a managed console interface that does things like rendering and that stuff for you. We might want to Discuss the header structure in the Future, but I think that we should wait for that until the important features are implemented. I have moved it into #192.

Reply:
Yes I agree this can wait for sure